code/game/objects/structures/kitchen_spike.dm

```diff
@@ -55,10 +68,14 @@
 	if(..())
 		return
 	if(src.occupied)
-		if(src.meattype == 1)
-			if(src.meat > 1)
+		if(src.meattype == MEATTYPE_MONKEY && src.skintype == SKINTYPE_MONKEY)
+			if(src.skin == 1)

```

Comment:
Mmmm, actually, Miauw is right. If an admin edits the spiked monkey to have more than 1 skin, then it should be >=